lib/widgets/components/forms/forgot_passcode/forgot_passcode_form.dart

```diff
@@ -53,15 +53,13 @@ class ForgotPasscodeForm extends StatelessWidget {
 
     final either = await sl<AccountRepository>().requestPasscodeReset(email);
 
-    var title = Strings.forgotPasscodeLinkSent;
-    var body = Strings.forgotPasscodeSentRequestTo(email);
-
-    either.fold(
-      (l) {
-        title = Strings.forgotPasscodeError;
-        body = l.message;
-      },
-      (r) => null,
+    final title = either.fold(
+      (_) => Strings.forgotPasscodeError,
+      (_) => Strings.forgotPasscodeLinkSent,
```

Comment:
For the sake of readability, we might want to use a name for the variables, instead of _, even though we are not using the values. Right now it is not apparent which of the cases is the error and success, unless you memorized the syntax of fold